### ISSUE TITLE
Improve Samples in Media

### ISSUE THREAD

**Issue by Arlodotexe** [CommunityToolkit/Windows#124]

**Comment by Arlodotexe** [comment]
In https://github.com/CommunityToolkit/Windows/pull/151, we ported all existing samples for the Media package from the 7.x sample app. This included samples for Shadows and EffectAnimations, but not for Brushes, Effects, or Pipelines. 

We have users asking for samples on how to use things like TilesBrush, and so we should add them to the gallery. Additionally, for components without tests, this gives us a simple way to ensure these tools work as expected.

I've reopened this issue so we can add samples for the following:
- [ ] Brushes
- [ ] Effects
- [ ] Pipelines

**Comment by michael-hawker** [comment]
FYI @TheJoeFin posted to [ask about TilesBrush on BlueSky here](https://bsky.app/profile/thejoefin.com/post/3ld5hjub4lc2n) - seems like it may be broken?

@Arlodotexe since we're probably not shipping 8.2 now until the new year, this could be a thing to spend a day and prop up again on our radar for any clean-up leading into it?

**Comment by michael-hawker** [comment]
Old Sample Pages that may have some of this code to move: https://github.com/CommunityToolkit/WindowsCommunityToolkit/tree/main/Microsoft.Toolkit.Uwp.SampleApp/SamplePages

**Comment by michael-hawker** [comment]
@erinpwoo is working on some improvements here.

**Comment by erinpwoo** [comment]
Almost done making the last set of changes for adding brush and pipeline samples. Will publish PR by EOD 2/21 @michael-hawker

**Comment by DaveAurionix** [comment]
@erinpwoo if you have a sample for the TilesBrush coming as part of the brush samples you mention, that would be awesome.  I'm asking because in the latest stable package (`8.1.240916`) the TilesBrush appears to be broken (seems not to render anything).  Same in latest preview package `8.2.250129-preview2`.  The sample would provide evidence that it basically works and would help me spot what I'm doing differently and raise a targeted issue and repro steps if needed.

**Comment by erinpwoo** [comment]
@DaveAurionix yes, a TilesBrush sample is included. Link to PR: https://github.com/CommunityToolkit/Windows/pull/639

**Comment by michael-hawker** [comment]
@erinpwoo have you switched to check WinUI 3 or just been running against UWP while developing?

**Comment by Arlodotexe** [comment]
> In [#151](https://github.com/CommunityToolkit/Windows/pull/151), we ported all existing samples for the Media package from the 7.x sample app. This included samples for Shadows and EffectAnimations, but not for Brushes, Effects, or Pipelines.
> 
> We have users asking for samples on how to use things like TilesBrush, and so we should add them to the gallery. Additionally, for components without tests, this gives us a simple way to ensure these tools work as expected.
> 
> I've reopened this issue so we can add samples for the following:
> 
> - [ ]  Brushes
> - [ ]  Effects
> - [ ]  Pipelines

Samples for Pipelines and Brushes were added in #654, meaning only Effects samples are left to add.
